lib/Elastica/Multi/Search.php

```diff
@@ -78,8 +74,7 @@ public function clearSearches(): self
     }
 
     /**
-     * @param BaseSearch $search
-     * @param string     $key
+     * @param string $key
```

Comment:
This is not correct IMHO, as it should be `string|null`

Reply:
maybe we should add `?string` in the type-hint of the method and remove the php-doc for the param?
Even better we could also use the `nullable_type_declaration_for_default_null_value` option for PHP-CS-FIXER to enforce the rule, WDYT?

Reply:
In this case I believe it is an error on the docblock: we accept null, but the annotation not.
The annotation is more strict than the method signature.

I'd rather (1) remove the annotation (if we handle null values), or (2) check the implementation if null values are not accepted. If (2) the method signature is wrong, and then the annotation is superfluous :)

Reply:
https://github.com/php/php-src/pull/3535

Reply:
Didn't know about `nullable_type_declaration_for_default_null_value` and the discussion. LGTM to use it :+1: 

Anyway: is the method really accepting null values?

Reply:
Null values seem to be accepted, so I'm going to remove the annotation, as argument type-hint is sufficient